pycoal/tests/mineral_test.py

```diff
@@ -81,7 +82,36 @@ def test_classify_image():
         # verify that every pixel has the same classification
         assert numpy.array_equal(expected.asarray(), actual.asarray())
 
+# verify that classified images have valid classifications when using config file
+# three basic tests w/ diff parallel methods and loading image in mem
+@with_setup(None, _test_classify_image_teardown)
+def test_classify_image_dask():
+    # set our config file to dask
+    config = configparser.ConfigParser()
+    config.read('config.py')
+    config['parallel_method'] = 'dask'
+    test_classify_image()
+    test_classify_image_in_memory()
+    
+@with_setup(None, _test_classify_image_teardown)
+def test_classify_image_pytorch():
+    # set our config file to pytorch
```

Comment:
Your not setting the config file, you are setting the `[processing][impl]` config key.

Reply:
updated